dagshub/data_engine/model/metadata_field_builder.py

```diff
@@ -100,6 +160,12 @@ def _remove_tag(self, tag: str):
         except ValueError:
             logger.warning(f"Tag {tag} doesn't exist on the field, nothing to delete")
 
+    def _remove_tags(self, tags: Set[str]):
+        if self.schema.tags is None:
+            return
+        for t in tags:
+            self._remove_tag(t)
```

Comment:
Change the signature
```suggestion
    def _remove_tags(self, *tags: str):
        if self.schema.tags is None:
            return
        for t in tags:
            self._remove_tag(t)
```

And in the usages removes the curly braces